### DISCUSSION TITLE
pandasai example code, no openai api key error

### DISCUSSION THREAD

**Discussion by ba703863** [sinaptik-ai/pandas-ai#57] [Q&A] [answered]
Can anyone write a sample code that can input the openai api key for the pandasai project

**Comment by gventuri** [comment, marked as answer]
Hey @ba703863, the API key is personal and should never be shared.

Check it out: https://help.openai.com/en/articles/4936850-where-do-i-find-my-secret-api-key

Closing, feel free to reopen if you have any questions
